src/cloud-api-adaptor/test/provisioner/trustee_kbs.go

```diff
@@ -29,20 +29,33 @@ import (
 const TRUSTEE_REPO_PATH = "../trustee"
 
 func getHardwarePlatform() (string, error) {
-	out, err := exec.Command("uname", "-i").Output()
+	out, err := exec.Command("uname", "-m").Output()
 	return strings.TrimSuffix(string(out), "\n"), err
 }
 
+func getOverlaysPath() (string, error) {
+	arch, err := getHardwarePlatform()
```

Comment:
nit: s/arch/platform to align with the method name and other usages later in the code

Reply:
Done. Thanks